src/Nethermind/Nethermind.Trie/Pruning/OverlayTrieStore.cs

```diff
@@ -24,4 +23,7 @@ public override TrieNode FindCachedOrUnknown(Hash256? address, in TreePath path,
 
     public override byte[]? TryLoadRlp(Hash256? address, in TreePath path, Hash256 hash, ReadFlags flags = ReadFlags.None) =>
         base.TryLoadRlp(address, in path, hash, flags) ?? store.TryLoadRlp(address, in path, hash, flags);
+
+    // TODO clarify is ClearCache is reliable enough to use
+    public void ResetOverrides() => ClearCache();
```

Comment:
Comment under `ClearCache`  says that it's added "to support testing", but it seems like a good way to reset changes applied on top of `OverlayTrieStore` from state overrides.

Reply:
I dont see a problem.